core/src/test/java/nl/nn/adapterframework/http/HttpSenderTest.java

```diff
@@ -141,6 +144,31 @@ public void simpleMockedHttpGet() throws Throwable {
 		}
 	}
 
+	@Test
+	public void simpleMockedHttpGetEncodeMessage() throws Throwable {
+		HttpSender sender = getSender(false); //Cannot add headers (aka parameters) for this test!
+		Message input = new Message("this is my dynamic url");
+
+		try {
+			IPipeLineSession pls = new PipeLineSessionBase(session);
+
+			sender.setMethodType("GET");
+			sender.setEncodeMessages(true);
+
+			sender.configure();
+			sender.open();
+
+			String result = sender.sendMessage(input, pls).asString();
+			assertEquals(getFile("simpleMockedHttpGetEncodeMessage.txt"), result.trim());
+		} catch (SenderException e) {
+			throw e.getCause();
+		} finally {
+			if (sender != null) {
+				sender.close();
+			}
+		}
+	}
+
```

Comment:
Ik snap niet wat je hier test / hoe deze test werkt:
Je stopt er onzin in, er moet een specifieke file uit komen.